ports/esp32/modules/inisetup.py

```diff
@@ -33,7 +33,7 @@ def fs_corrupted():
 def setup():
     check_bootsec()
     print("Performing initial setup")
-    uos.VfsFat.mkfs(bdev)
+    uos.VfsLfs2.mkfs(bdev)
     vfs = uos.VfsFat(bdev)
```

Comment:
Shouldn't this be 

```
vfs = uos.VfsLfs2(bdev)
```

as in the commit for esp8266?

Reply:
Indeed.  Now fixed.